underfs/glusterfs/src/test/java/tachyon/underfs/glusterfs/GlusterFSUnderFileSystemFactoryTest.java

```diff
@@ -41,9 +41,8 @@
   @Before
   public final void before() throws IOException {
     mTachyonConf = new TachyonConf();
-    mMount =  mTachyonConf.get(Constants.UNDERFS_GLUSTERFS_MR_DIR,
-        "glusterfs:///mapred/system");
-    mVolume = mTachyonConf.get(Constants.UNDERFS_GLUSTERFS_VOLUMES, null);
+    mMount =  mTachyonConf.get(Constants.UNDERFS_GLUSTERFS_MR_DIR);
```

Comment:
e.g., since `tachyon.underfs.glusterfs.mapred.system.dir` is not set in `tachyon-default.properties`, you may need to define it in `tachyon-default.properties`